ts/features/itwallet/walletInstance/store/reducers/index.ts

```diff
@@ -27,10 +28,18 @@ const reducer = (
   switch (action.type) {
     case getType(itwWalletInstanceAttestationStore): {
       return {
+        ...state,
```

Comment:
I think it's correct to reset the state when we save a new WI: if there is already a `status` stored, it is for sure referred to a past WI.
What do you think?